.storybook/preview.tsx

```diff
@@ -1,10 +1,12 @@
-import type { Preview } from '@storybook/react'
+import { type Preview } from '@storybook/react'
 
 import { SeamProvider } from 'index.js'
 
 // UPSTREAM: Use toolbar input when it lands.
 // https://github.com/storybookjs/storybook/pull/21959
-const publishableKey = 'seam_pk1fGd41X_zKs0ZELRWEc8nWxiBsrTFC98'
+const defaultPublishableKey =
+  process.env['STORYBOOK_SEAM_PUBLISHABLE_KEY'] ??
+  'seam_pk1fGd41X_zKs0ZELRWEc8nWxiBsrTFC98'
```

Comment:
what's the default here when env var not set? should we just error instead if not set?

Reply:
This is for zero-setup local development. The env var is used as an override option or in deployment. Eventually there will be in input box in storybook for it.